src/ClockKit/CLKComplication.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using ObjCRuntime;
+
+namespace ClockKit {
+	public partial class CLKComplication {
+			[Watch (7,0)]
+			[DllImport (Constants.ClockKitLibrary, EntryPoint = "CLKAllComplicationFamilies")]
+			public static extern NSNumber[] GetAllComplicationFamilies ();
```

Comment:
oh, that needs a test because it defy my understanding of p/invokes wrt ObjC ;-)